tools/packaging/release/tag_repos.sh

```diff
@@ -150,9 +149,13 @@ tag_repos() {
 
 push_tags() {
 	info "Pushing tags to repos"
-	build_hub
+	build_gh
```

Comment:
Isn't easier to just download gh from https://github.com/cli/cli/releases/tag/v2.37.0?

Reply:
This could avoid the problem of requiring a too recent golang to do the build indeed. I'll have a look.